src/main/java/nl/beehivejava/scriptj/load/PluginLoader.java

```diff
@@ -0,0 +1,13 @@
+package nl.beehivejava.scriptj.load;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+/**
+ * @author Lesley
+ */
+public interface PluginLoader {
+
+    InputStream load(String location) throws IOException;
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AUndocumentedApi)